src/poppler-qt5.mk

```diff
@@ -0,0 +1,23 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := poppler-qt5
+$(PKG)_WEBSITE  := $(poppler_WEBSITE)
+$(PKG)_IGNORE   := $(poppler_IGNORE)
+$(PKG)_VERSION  := $(poppler_VERSION)
+$(PKG)_CHECKSUM := $(poppler_CHECKSUM)
+$(PKG)_SUBDIR   := $(poppler_SUBDIR)
+$(PKG)_FILE     := $(poppler_FILE)
+$(PKG)_URL      := $(poppler_URL)
```

Comment:
I think all these need to be expanded variables i.e `=` instead of `:=`.

Reply:
Done.